src/Blockcore.Indexer.Core/Storage/Mongo/MongoData.cs

```diff
@@ -1495,31 +1495,44 @@ public async Task<List<OutspentResponse>> GetTransactionOutspendsAsync(string tx
 
       public async Task<OutspentResponse> GetOutputSpendingStatusAsync(Outpoint outpoint)
       {
-         // check if the output is spent in the mempool
          var spentOutput = (await mongoDb.InputTable.FindAsync(i => i.Outpoint == outpoint)).FirstOrDefault();
 
          OutspentResponse response = new()
          {
             Spent = false,
+            Txid = null,
```

Comment:
if we actually want to return the same result as in mempool (when not spent the parameters dont exist) we need to create a new type or put an attribute to no serialize null properties